pkg/pipeline/PropertiesConfig.go

```diff
@@ -277,17 +279,20 @@ func (impl PropertiesConfigServiceImpl) CreateEnvironmentProperties(appId int, e
 		IsOverride:        envOverride.IsOverride,
 	}
 
-	chartMajorVersion, chartMinorVersion, err := util2.ExtractChartVersion(chart.ChartVersion)
 	if err != nil {
 		impl.logger.Errorw("chart version parsing", "err", err, "chartVersion", chart.ChartVersion)
 		return nil, err
 	}
-
-	if !(chartMajorVersion >= 3 && chartMinorVersion >= 1) {
+	chartRefValue, err := impl.chartRefRepository.FindById(environmentProperties.ChartRefId)
+	if err != nil {
+		impl.logger.Errorw("error in finding ref chart by id", "err", err)
+		return nil, err
+	}
+	if !(chartRefValue.IsAppMetricsSupported) {
 		appMetricsRequest := chartService.AppMetricEnableDisableRequest{UserId: environmentProperties.UserId, AppId: appId, EnvironmentId: environmentProperties.EnvironmentId, IsAppMetricsEnabled: false}
 		_, err = impl.EnvMetricsEnableDisable(&appMetricsRequest)
 		if err != nil {
-			impl.logger.Errorw("err while disable app metrics for lower versions", "err", err, "appId", appId, "chartMajorVersion", chartMajorVersion, "chartMinorVersion", chartMinorVersion)
+			impl.logger.Errorw("err while disable app metrics for lower versions", "err", err, "appId", appId, "chartVersion", chart.ChartVersion)
```

Comment:
change error message, you can remove lower versions.

Reply:
Implemented 👍